packages/apollo-language-server/src/providers/schema/endpoint.ts

```diff
@@ -29,6 +29,17 @@ export class EndpointSchemaProvider implements GraphQLSchemaProvider {
       uri: url,
       fetch
     };
+
+    if (!url) {
+      throw new Error(
```

Comment:
Could we throw a CLIError here to get proper formatting? :)

Reply:
Resolving this as `CLIError` is only available in the CLI code. I do think this error message will never trigger, though, because the default is to make `--endpoint` localhost:4000